### DISCUSSION TITLE
運動&AI

### DISCUSSION THREAD

**Discussion by Timmy110403540** [1101-GS3073-C/Python-Community-GS3073-C#74] [科技新知]
之前老師有發了一篇有關於分析籃球的AI文章,我找了一篇也是有關於人工智慧在運動方面的發展
從之前熟悉的AlphaGo打敗棋王後 人工智慧一直於各方面蓬勃發展 運動也不例外
文章中列舉了幾點AI的重要貢獻:做為一個虛擬教練培訓,幫助選手,或是在場上分析進球率,分析該如何進攻或防守等等.
又抑或是當作裁判,像是在奧運中精采的羽球男雙決賽的最後一球,使用了鷹眼(hawk-eye)系統,準確地判斷落球位置(in!)
文章中也在最後出現老師分享的Nex Team系統 歡迎點入一窺AI及運動間巧妙的連結!

文章來源:https://findit.org.tw/researchPageV2.aspx?pageId=1770

**Comment by chia-kai-chang** [comment]
鷹眼(hawk-eye)系統真的很厲害，馬上就可以從多個攝影機的影像資料還原賽事現場!

**Reply by Timmy110403540** [reply]
是的 很欣賞這項技術!!

**Comment by Jay-722** [comment]
準確判斷位置真的很屌，超讚的。

**Comment by jim11261314** [comment]
未來有更精彩的運動捕捉即時畫面
終於不會錯過鄧肯偷笑被罰的鏡頭了

**Comment by Yunhans** [comment]
學校球場什麼時候可以有鷹眼呢
